### DISCUSSION TITLE
Versions 2.0.0 and 2.1.0 in Container Registry

### DISCUSSION THREAD

**Discussion by anilsaldhana** [microsoft/FHIR-Converter#223] [Q&A: Handlebars Engine]
We have had the fhirconverter app service deployed in Azure for a few months now. We are using v2.0.0.  

When the container was restarted recently, we started seeing docker image pull errors similar to the following:
`2021-03-25T16:29:02.022Z ERROR - DockerApiException: Docker API responded with status code=NotFound, response={"message":"manifest for healthplatformregistry.azurecr.io/fhirconverter:v2.1.0 not found: manifest unknown: manifest tagged by \"v2.1.0\" is not found"}`

This message is the same for both v2.0.0 and v2.1.0 tags of the project. 

Did anything change with the container registry?  Was the CR decommissioned? 
Did the specific versions 2.0.0 and 2.1.0 of this project deleted from the Container Registry?

Looking forward to your guidance.

**Comment by ranvijaykumar** [comment]
+ @qiwjin

**Reply by ranvijaykumar** [reply]
Can you please verify the behavior now? We have applied the fix for this issue.

**Comment by mjohnson-xbridge-io** [comment]
Looking for guidance on this, as well. We had a similar issue in December as I wrote about https://github.com/microsoft/FHIR-Converter/discussions/137. This appears to have happened again. Is there guidance on how to install a single version of the app service and keep it stable?

I can see our templates using ftp. However, they do not appear in the service.

**Reply by ACMoretxj** [reply]
Are you still have this issue after restarting the app service? (make sure your templates are backed up before restarting for safety's sake).

**Comment by anilsaldhana** [comment]
The issue has been resolved. Can you please share what the root cause of
this issue was?

We really appreciate it.
